sdks/java/io/hadoop-common/pom.xml

```diff
@@ -64,6 +64,16 @@
       <artifactId>jsr305</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
I understand why you added the dependency. However, you should add the dependency in a different place - where it is used in the io/hadoop module, not the io/hadoop-common module.